lib/CommandImplementations.ts

```diff
@@ -164,7 +173,11 @@ export async function handleGen3Count(msg: Message, args: string): Promise<void>
     const res = await fetch(url);
 
     const address = args.trim();
-  
+    if (address != '' && !isValidSolAddress(address)) {
```

Comment:
```suggestion
    if (address !== '' && !isValidSolAddress(address)) {
```

https://stackoverflow.com/a/359509/8737306